Source/Example/Scenarios/S04_Write_to_stream.cs

```diff
@@ -58,6 +60,40 @@ void WriteSequentiallyToExistingStream()
             }
         }
 
+        void WriteMultipleStreamsInParallel()
+        {
+            const int streamsToWrite = 10;
+
+            Enumerable.Range(1, streamsToWrite).AsParallel()
+                .ForAll(streamIndex =>
+                {
+                    var partition = new Partition(Partition.Table, streamIndex.ToString());
+
+                    var existent = Stream.TryOpen(partition);
+
+                    var stream = existent.Found
+                        ? existent.Stream
+                        : new Stream(partition);
+
+                    Console.WriteLine("Writing to new stream in partition '{0}'", partition);
+                    var stopwatch = Stopwatch.StartNew();
+
+                    for(int i=0; i<30; ++i)
```

Comment:
Can I ask you for a minor change here? Make spaces in `for` statement consistent with the rest of code.  I need to push my ReSharper settings file along with solution! :smile: